### DISCUSSION TITLE
Composer integration ?

### DISCUSSION THREAD

**Discussion by hpalavinch** [fabianwennink/IconCaptcha-Plugin-jQuery-PHP#32] [Q&A] [answered]
Hi fabianwennink

I was just wandering if you would consider a composer packagist integration for your library which looks great

I encounter difficulties to integrate the library in the assets of the project i work on, as they are not public.
The home made framework which is used for the web app blocks the call to captcha-request.php (for validation) as the resource does not have a public access

So using it with composer  would be really great

**Comment by fabianwennink** [comment, marked as answer]
Hi @hpalavinch! This is currently in development and planned to be released somewhere in the next couple of months. See issue https://github.com/fabianwennink/IconCaptcha-Plugin-jQuery-PHP/issues/14, discussion https://github.com/fabianwennink/IconCaptcha-Plugin-jQuery-PHP/discussions/29 and the [develop branch](https://github.com/fabianwennink/IconCaptcha-Plugin-jQuery-PHP/tree/develop). However, note that the composer package will have to make calls to an endpoint to get the challenge and validate the input. So if the request is currently blocked, it will still be blocked even with the composer package.

**Reply by hpalavinch** [reply]
Actually as the hand-made framework is, if the lib is in the vendor directory, it may work. I hope so !!
Thanks for your quick answer
